test/e2e/reconciler/workspace/controller_test.go

```diff
@@ -171,6 +171,8 @@ func TestWorkspaceController(t *testing.T) {
 		},
 	}
 
+	server := framework.SharedKcpServer(t)
```

Comment:
line 77 is potentially disruptive. Have you checked that it isn't, e.g. by not calling t.parallel? I don't think this shared fixture will work otherwise.

Reply:
Oof. I'll submit it separately then.

Reply:
Removed for now.